### PR TITLE
chore(operator): determine version of postgresql

### PR DESCRIPTION
**Pull request by zregvart**
This the backport of #7986 for 1.9.x

### CODE REVIEW

**Comment by zregvart** [comment]
Fixed `assets_vfsdata.go` (hopefully)

**Comment by zregvart** [comment]
I'm going to rebase this now.

**Comment by zregvart** [comment]
Note the inclusion of changes from #7983 in e482fb3

File: install/operator/pkg/syndesis/configuration/configuration.go [hunk 1]
```diff
@@ -281,38 +293,92 @@ func GetProperties(file string, ctx context.Context, client client.Client, synde
...
+		for _, c := range databaseDeployment.Spec.Template.Spec.Containers {
+			if c.Name == "postgresql" {
+				//
+				// Does deploment config already contain UPGRADE Env Var?
+				// if it does then DO NOT remove it.
+				//
+				for _, env := range c.Env {
+					if env.Name == "POSTGRESQL_UPGRADE" {
+						configuration.DatabaseNeedsUpgrade = true
+						return configuration, nil
```
Comment (phantomjinx) [inline install/operator/pkg/syndesis/configuration/configuration.go]: That works nicely
+1

**Comment by zregvart** [comment]
retest this please

**Comment by zregvart** [comment]
retest this please

**Comment by avano** [comment]
@zregvart seems like the `syndesis-circleci-to-quay` failed for the UI and operator images as they not present in quay: 

```
docker pull quay.io/syndesispr/syndesis-ui:7990-f657bf7a8584d538940b1ab7a32c58192041285e                                                                                                           
Error response from daemon: manifest for quay.io/syndesispr/syndesis-ui:7990-f657bf7a8584d538940b1ab7a32c58192041285e not found: manifest unknown: manifest unknown

docker pull quay.io/syndesispr/syndesis-operator:7990-f657bf7a8584d538940b1ab7a32c58192041285e
Error response from daemon: manifest for quay.io/syndesispr/syndesis-operator:7990-f657bf7a8584d538940b1ab7a32c58192041285e not found: manifest unknown: manifest unknown
```

**Comment by avano** [comment]
I rebased on top of 1.9.x

**Comment by zregvart** [comment]
Thanks @avano

**Pull request merged**